plugins/fields.yaml

```diff
@@ -0,0 +1,70 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: fields
+spec:
+  platforms:
+  - uri: https://github.com/rewanth1997/kubectl-fields/releases/download/v1.2.0-beta/kubectl-fields-Darwin-x86_64.tar.gz
+    sha256: 35072754f24a0bf239c6bb90a53c528b7b88f99435e2a142907599b18169fade
+    bin: kubectl-fields
+    files:
+    - from: "*"
+      to: "."
+    selector:
+      matchLabels:
+        os: darwin
+        arch: amd64
+  - uri: https://github.com/rewanth1997/kubectl-fields/releases/download/v1.2.0-beta/kubectl-fields-Linux-x86_64.tar.gz
+    sha256: 730602c8400ad7b712a2eab26d3e1a5feb6593f442f369a3ea246fbb37f6fd3d
+    bin: kubectl-fields
+    files:
+    - from: "*"
+      to: "."
+    selector:
+      matchLabels:
+        os: linux
+        arch: amd64
+  - uri: https://github.com/rewanth1997/kubectl-fields/releases/download/v1.2.0-beta/kubectl-fields-Windows-x86_64.zip
+    sha256: 2ea706370ea3260e9e68943947bbaff295bc4fa1b97cf1492912b01d04d303a2
+    bin: kubectl-fields.exe
+    files:
+    - from: "*"
+      to: "."
+    selector:
+      matchLabels:
+        os: windows
+        arch: amd64
+  version: v1.2.0-beta
+  homepage: https://github.com/rewanth1997/kubectl-fields
+  caveats: |
+    Usage:
+      $ kubectl fields <RESOURCE> [options] <PATTERN>
+
+    Options:
+      -I, --case-sensitive   Case sensitive pattern match
+      -h, --help             help for kubectl-fields
+          --no-color         Do not print colored output
+          --stdin            Expects input via pipes
+
+    Read the documentation:
+      https://github.com/rewanth1997/kubectl-fields/tree/v1.2.0-beta
+  shortDescription: Grep resources hierarchy by field name
+  description: |
+    kubectl-fields parses specified kubectl resources to match given pattern(s).
+    It prints matched fields parental hierarchy in one-liner format.
+
+    More on https://github.com/rewanth1997/kubectl-fields
+
+    Examples:
+
+    Find resource field names:
+    $ kubectl fields po.spec capa
+    containers.securityContext.capabilities
+    initContainers.securityContext.capabilities
+
+    Find resource field names case-insensitively:
+    $ kubectl fields svc -i affinity
+    spec.sessionAffinity
+    spec.sessionAffinityConfig`
+
+    No more greps, pipes and scrolling up and down to find hierarchial order for resources :-)
```

Comment:
I think this last line doesn't add much.
It's pretty clear from above what it does.

Also perhaps move examples to `-h`/`--help` of your program instead.

Reply:
Adding more examples increases cluttering in the terminal as @corneliusweig mentioned earlier. Hence I added minimal examples. I added more examples as screenshots to the README [examples](https://github.com/rewanth1997/kubectl-fields#examples) section.

Please share the other apt examples to be added to the help menu in your opinion.